### ISSUE TITLE
Waffle column API can give an inaccurate number of issues

### ISSUE THREAD

**Issue by espadrine** [waffleio/waffle.io#2466]
Hello!

As part of an effort to offer shields.io badges for Waffle.io, [@evancohen](https://github.com/badges/shields/issues/257#issuecomment-203697850) noticed that the API sometimes gives inaccurate information. For instance, the following command:

``` bash
curl 'https://api.waffle.io/evcohen/smart-mirror/columns'
```

… returns `"issues": []` for the “_in progress_” label, even though there is one: `curl 'https://api.github.com/repos/evancohen/smart-mirror/issues?labels=in%20progress' | jq length` → 1.

In the meantime, I will try to offer a GitHub badge.

Thanks for working on this great platform!

Related issues: #468, #2233.

**Comment by homeyer** [comment]
Hey @espadrine!

We've deprecating the `issues[]` collection on columns, just haven't removed the data entirely yet.

There isn't a quick way to just get the count of cards in a column, but you can fetch cards with `https://api.waffle.io/:owner/:repo/cards` and look at the `githubMetadata.labels` to determine which column it fits in.

**Comment by evancohen** [comment]
I've attempted to use the `/cards` endpoint to generate shields, but there are some limitations: badges/shields#681

tl;dr You can't get the color for columns that have 0 cards and you can't get a count for backlog/done/archive. It would be great to see a simple endpoint that would surface this data.

**Comment by homeyer** [comment]
Long time coming, but we're finally updating our badges to match shield styling. We'll also offer new badges soon that show all the columns for a board.

There's a new API endpoint `https://api.waffle.io/waffleio/waffle.io/columns?with=count` to see counts for each column for any public waffle boards.
